README.md

```diff
@@ -13,8 +13,10 @@ Download the PDF here:
 I've built the mindmap with MindNode for the Mac. https://mindnode.com
 
 ## Companion Notebook
-This Mindmap/Cheatsheet has a companion Jupyter Notepad that runs through most of the Data Science steps that can be found at the following link:
+This Mindmap/Cheatsheet has a companion Jupyter Notebook that runs through most of the Data Science steps that can be found at the following link:
 - https://github.com/dformoso/sklearn-classification
+Jupyter Notebook installation guide:
```

Comment:
I rather not include this as the companion notebook has the jupyter install configured as a docker container, and some people might get confused, as in: Do they need to install Jupyter or is the Docker install enough?